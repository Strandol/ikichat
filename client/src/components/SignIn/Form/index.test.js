import { shallow } from 'enzyme'
import React from 'react'

import { WrappedForm } from './index'

describe('SignInForm', () => {
    it('render', () => {
        const c = shallow(<WrappedForm />)

        expect(c).toMatchSnapshot()
    })
})
