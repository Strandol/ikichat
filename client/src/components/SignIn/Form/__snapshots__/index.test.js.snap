// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInForm render 1`] = `ShallowWrapper {}`;
